components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/security/handler/AuthorizationHandler.java

```diff
@@ -82,12 +82,24 @@ protected Boolean authorize(String userName) {
             }
         } else {
             //Uses in memory user store
-            LOG.error("Authorization is not supported with the in memory user store. Please plug in a user store for "
-                      + "the correct functionality");
-            return false;
+            return processAuthorizationWithFileBasedUserStore(userName);
         }
     }
 
+    /**
+     * Processes the authorization request using file based user store.
+     *
+     * @return true if successfully authorized
+     */
+    private boolean processAuthorizationWithFileBasedUserStore(String userName) {
+
+        boolean isAuthorized = FileBasedUserStoreManager.getUserStoreManager().isAdmin(userName);
```

Comment:
here, the logic is we are authorizing if that is an admin, right? In that case, shall we rename isAuthorized ti isAdmin, else code is not readable.

Reply:
Fixed with df6ab2b